udata/templates/user/base.html

```diff
@@ -34,14 +34,14 @@
 {% block main_content %}
 <div class="row">
     <div class="col-md-3">
-        <div class="panel panel-default" itemscope itemtype="http://schema.org/Person">
+        <div class="panel panel-default">
             <div class="panel-body">
                 <div class="text-center">
                     <img src="{{ user.avatar|placeholder('user') }}"
-                        alt="{{ user.fullname }}" class="scalable" itemprop="image"/>
+                        alt="{{ user.fullname }}" class="scalable"/>
                 </div>
-                <h2 itemprop="name">{{ user.fullname }}</h2>
-                <div itemprop="description">
+                <h2>{{ user.fullname }}</h2>
+                <div>
                     {{ user.about|markdown }}
                 </div>
                 {# <hr/> #}
```

Comment:
Yes, we need to check, but I think we can remove a lot of useless divs which were present only for microdata scoping.